cirq/google/devices/known_devices.py

```diff
@@ -279,3 +279,28 @@ def _json_dict_(self):
 Sycamore = SerializableDevice.from_proto(
     proto=SYCAMORE_PROTO,
     gate_sets=[gate_sets.SQRT_ISWAP_GATESET, gate_sets.SYC_GATESET])
+
+
+# Subset of the Sycamore grid with a reduced layout.
+_SYCAMORE23_GRID = """
+----------
+----------
+----------
+--A-------
+-ABC------
+ABCDE-----
+-CDEFG----
+--EFGHI---
+---GHI----
+----I-----
+"""
+
+SYCAMORE23_PROTO = create_device_proto_from_diagram(
+    _SYCAMORE_GRID,
```

Comment:
Pylint didn't catch that `_SYCAMORE23_GRID` wasn't used?
```suggestion
    _SYCAMORE23_GRID,
```